tests/test_utils/test_textio.py

```diff
@@ -32,7 +69,22 @@ def test_list_to_file():
             assert all(line1 == line2 for line1, line2 in zip(lines, lines2))
 
 
-def test_list_from_file():
+def test_list_to_file_json():
```

Comment:
Since we are just testing a single function, we can put all the tests into `test_list_to_file` and add comments before each section.